workloads/kube-burner/common.sh

```diff
@@ -136,3 +153,19 @@ cleanup() {
   oc delete ns -l kube-burner-uuid=${UUID}
 }
 
+get_pprof_secrets() {
+ export certkey=`oc get secret -n openshift-etcd | grep "etcd-serving-ip" | head -1 | awk '{print $1}'`
```

Comment:
You don't need to export this one
```suggestion
 local certkey=`oc get secret -n openshift-etcd | grep "etcd-serving-ip" | head -1 | awk '{print $1}'`
```